### DISCUSSION TITLE
http://localhost:8080/guides/animation/simple-texture-animation-in-blender

### DISCUSSION THREAD

**Discussion by giscus** [morrowind-modding/morrowind-modding.github.io#63] [General]
# http://localhost:8080/guides/animation/simple-texture-animation-in-blender

Make a texture orbit around a sphere in this short Blender tutorial.

http://localhost:8080/guides/animation/simple-texture-animation-in-blender

**Comment by magicaldave** [comment]
this guide gave me corprus
